notebooks/03_basic_preprocessing_categorical_variables.ipynb

```diff
@@ -287,7 +287,7 @@
     "lines_to_next_cell": 2
```

Comment:
<p>This is a test comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lesteve/scikit-learn-tutorial/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>